src/lwc/utilExpandableSection/utilExpandableSection.js

```diff
@@ -54,6 +55,9 @@ export default class utilExpandableSection extends LightningElement {
 
     toggleSection() {
         this.isCollapsed = !this.isCollapsed;
+        if (this.shouldInformParent) {
```

Comment:
High likelihood that this will be true given that the parent component populates the public property `should-inform-parent` with 'false' since that string would be truthy.

Might suggest doing a coercive equality check `this.shouldInformParent == false` or create a public setter for the `shouldInformParent` property and perform the coercion there.

Reply:
Fixed!